src/customers/dto/customer.indto.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export default class CustomerInDTO {
```

Comment:
Are we prone to errors? What happens if I send this body?

```
{
    "firstName": null,
    "lastName": "Spring",
    "username": "JavaScript5",
    "password": "youllneverguess2",
    "emailAddress": "email@address.com"
}
```

💥 Internal server error 💥

If you have time, try at least for a DTO to include `class-validator` to ensure that the payload is correct and the wrong one is handled.

Reply:
done